core/write.go

```diff
@@ -10,7 +10,24 @@ import (
 // the tag used to denote the name of the question
 const tagName = "survey"
 
+// add a few interfaces so users can configure how the prompt values are set
+type settable interface {
+	Set(value interface{}) error
```

Comment:
Of course we could use the same name as your function so require a `WriteAnswer(interface{}) error`